workspaces/frontend/src/app/pages/Workspaces/ExpandedDataVol.tsx

```diff
@@ -0,0 +1,37 @@
+import { Content } from '@patternfly/react-core';
+import { LockedIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+import { Workspace } from '~/shared/types';
+
+interface ExpandedDataVolProps {
+  workspace: Workspace;
+}
+
+export const ExpandedDataVol: React.FC<ExpandedDataVolProps> = ({ workspace }) => {
```

Comment:
Please consider renaming this component (and also other associated declarations).
```suggestion
export const DataVolumesList: React.FC<ExpandedDataVolProps> = ({ workspace }) => {
```